blog/views.py

```diff
@@ -11,8 +11,9 @@
 
 
 def course_list(request):
-    courses = Course.objects.filter(published_date__lte=timezone.now()).order_by('published_date').reverse()
-    return render(request, 'things/course_list.html', {'courses': courses})
+    current_courses = Course.objects.filter(archived=False).filter(published_date__lte=timezone.now()).order_by('published_date').reverse()
```

Comment:
Myslím že by mělo fungovat i:
```suggestion
    current_courses = Course.objects.filter(archived=False, published_date__lte=timezone.now()).order_by('published_date').reverse()
```